test/k8s/egress.go

```diff
@@ -76,8 +77,23 @@ var _ = SkipDescribeIf(func() bool {
 		res.ExpectSuccess()
 	}
 
-	BeforeAll(func() {
+	// ctEntriesOnGatewayNode returns the number of CT entries on the
+	// designated gateway node (k8s2) matching destination IP and port
+	ctEntriesOnGatewayNode := func(dstIP, dstPort string) int {
+		ciliumPodK8s2, err := kubectl.GetCiliumPodOnNode(helpers.K8s2)
```

Comment:
Maybe it makes sense to not have the function assume which is the gateway node, but rather leave it up to the caller to have that context?